### PR TITLE
Add script to collect debug data from PostgreSQL

### PR DESCRIPTION
**Pull request by jlpadilla**
### Related Issue
https://issues.redhat.com/browse/ACM-0000

### Description of changes
- Adds script to collect debug data from Postgres.

### CODE REVIEW

File: scripts/postgres-debug.sh [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+printf "\n>>> POSTGRESQL Configuration:\n\n"
+psql -d search -U searchuser -c "SHOW all;"
+
+printf "\n>>> POSTGRESQL Database size:\n\n"
+psql -d search -U searchuser -c "SELECT datname, pg_size_pretty(pg_database_size(datname)) from pg_database order by pg_database_size(datname) desc;"
+
+printf "\n>>> POSTGRESQL resource table size:\n\n"
+psql -d search -U searchuser -c "SELECT pg_size_pretty(pg_total_relation_size('search.resources'));"
+
```
Comment (SherinV) [inline scripts/postgres-debug.sh:45]:
```suggestion
printf "\n>>> POSTGRESQL edges table size:\n\n"
psql -d search -U searchuser -c "SELECT pg_size_pretty(pg_total_relation_size('search.edges'));"


```

Comment (jlpadilla) [reply scripts/postgres-debug.sh:45]: added

File: scripts/postgres-debug.sh [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+psql -d search -U searchuser -c "SELECT count(*) FROM search.edges;"
+
+printf "\n>>> Edge count by cluster:\n\n"
+psql -d search -U searchuser -c "SELECT cluster, count(*) FROM search.edges GROUP BY cluster ORDER BY count DESC;"
+
+printf "\n>>> Edge count by type:\n\n"
+psql -d search -U searchuser -c "SELECT edgetype, count(*) FROM search.edges GROUP BY edgetype ORDER BY count DESC;"
+
+printf "\n>>> Intercluster edge count by cluster:\n\n"
+psql -d search -U searchuser -c "SELECT cluster, count(*) FROM search.edges WHERE edgetype = 'intercluster' GROUP BY cluster ORDER BY count DESC;"
```
Comment (SherinV) [inline scripts/postgres-debug.sh:80]:
```suggestion
psql -d search -U searchuser -c "SELECT cluster, count(*) FROM search.edges WHERE edgetype = 'interCluster' GROUP BY cluster ORDER BY count DESC;"
```

Comment (SherinV) [reply scripts/postgres-debug.sh:80]: As per https://github.com/stolostron/search-v2-operator/blob/d71f894874c05740eee027a4dfaf6a89b71b0a62/controllers/create_pgconfigmap.go#L68. Have to change intercluster to interCluster

**Review by SherinV — APPROVED** [review]
/lgtm after the change in the comments.

**Pull request merged**